ref/__init__.py

```diff
@@ -23,4 +23,9 @@ def importer(name, *args, level: int = 0):
             except AttributeError: pass
     if level >= 1:
         pass # todo: add here some other functions
+    if restrictwrite == True:
```

Comment:
This was just copy-pasted from https://github.com/donno2048/restricted-functions/issues/1#issuecomment-863331876